system/settings/settings.xml

```diff
@@ -143,6 +143,17 @@
           <default>true</default>
           <control type="toggle" />
         </setting>
+        <setting id="winsystem.ishdrdisplay" type="boolean" label="13436" help="36299">
+          <dependencies>
+            <dependency type="visible">
+              <condition on="property" name="ishdrdisplay" />
+            </dependency>
+          </dependencies>
+          <!--requirement>ishdrdisplay</requirement-->
```

Comment:
> There is now a setting which is only visible if display supports HDR and let users switch between display HDR and internal GL(ES) shader HDR

 Really? If this commented out the setting will always be visible.

Reply:
nvm, sorry for the noise.

Reply:
Thx, I'll remove the comment - leftover from testing